RecoMTD/Configuration/python/RecoMTD_EventContent_cff.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+
+#FEVT
+RecoMTDFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        )
+)
+
+#RECO content
+RecoMTDRECO = RecoMTDFEVT.copy()
+
+#AOD content
+RecoMTDAOD = RecoMTDFEVT.copy()
+
+_phase2TimingLayer_EventContent = cms.untracked.vstring('keep *_trackExtenderWithMTD_*_*')
+
+from Configuration.Eras.Modifier_phase2_timing_layer_tile_cff import phase2_timing_layer_tile
+from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
+
+def modifyDataTier(DataTier):
+    added = DataTier.outputCommands + _phase2TimingLayer_EventContent
+    (phase2_timing_layer_tile | phase2_timing_layer_bar).toModify(DataTier, outputCommands = added)
```

Comment:
the eventContent setup appears to be a wrong place to control which modules are to be executed.
Please revert the last updates in this file and instead change the era modifications for ```globalreco``` in  Configuration/StandardSequences/python/Reconstruction_cff.py.